spec/factories/users.rb

```diff
@@ -0,0 +1,17 @@
+FactoryBot.define do
+  factory :user, class: 'User' do
+    first_name { Faker::Name.first_name }
+    last_name { Faker::Name.last_name }
+    patronymic { Faker::Name.middle_name }
+    salon_id { rand(1...101) }
```

Comment:
this should either create salon in after create hook, or should be provided on invocation

Reply:
Ok, i do it when associations will be ready